recipes/vulkan-validationlayers/all/patches/fix-cmake-1.2.182.patch

```diff
@@ -0,0 +1,14 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -237,10 +237,7 @@
+ option(BUILD_LAYER_SUPPORT_FILES "Generate layer files" OFF) # For generating files when not building layers
+ option(USE_ROBIN_HOOD_HASHING "Use robin-hood-hashing" ON)
+ if (USE_ROBIN_HOOD_HASHING)
+-    if(NOT ROBIN_HOOD_HASHING_INSTALL_DIR)
+-	set(ROBIN_HOOD_HASHING_INSTALL_DIR $ENV{ROBIN_HOOD_HASHING_INSTALL_DIR} PATH "Path to robin-hood-hashing repository")
+-    endif()
+-    set(ROBIN_HOOD_HASHING_INCLUDE_DIR "${ROBIN_HOOD_HASHING_INSTALL_DIR}/src/include" PATH "Path to robin-hood-hashing/src/include")
++    include_directories(${CONAN_INCLUDE_DIRS_ROBIN-HOOD-HASHING})
```

Comment:
We'll need to propose a PR upstream to avoid to patch ii forever.

Reply:
Yes, I second that. I think they assume people will clone the robin-hood-hash repo, thus the /src/include layout which breaks the build with the package from conan. The thing is that robin-hood-hash readme suggests either to use conan or to copy the only header file somewhere. I'll try and ask them what's the best way to solve this.